docs/guide/personalization/developer_guide/tracking_api.md

```diff
@@ -1,111 +1,173 @@
+If there's no existing user ID handling that we can re-use, it is recommended that you use your own cookie 
+and set the expiry date to at least 90 days from the last usage. 
+If there's a login mechanism, the user is usually tracked with a temporary identifier before the login. 
+Immediately after a successful login process a "login" event must be sent.
+At this point a pseudonymous user ID, for example, a system's internal registration id, must be used. 
+After logout, the anonymous user ID can be used again.
 
 !!! note
 
     The user identifier is required in tracking requests, otherwise it is discarded from the tracking servers.
 
-    If a browser has JavaScript or cookies disabled, please verify that you put some "dummy" value as identifier in the tracking request to avoid losing tracking information. Even if the event is not user-specific, it is still useful for popularity models (e.g. Topsellers, see [6. Recommendation Models](https://doc.ezplatform.com/projects/userguide/en/master/personalization/recommendation_models.md)).
+    If a browser has JavaScript or cookies disabled, make sure that you put some "dummy" value as identifier in the tracking request to avoid losing tracking information. 
+    Even if the event is not user-specific, it is still useful for [popularity models](https://doc.ibexa.co/projects/userguide/en/latest/personalization/recommendation_models/).
 
-Internally the recommendation engine creates a hash of every user ID. The original ID is not saved. It is still possible that the original ID appears in the log files for debug purposes but log files are are purged regularly. The user ID is case sensitive.
+The recommendation engine internally creates a hash of every user ID. 
+The original ID is not saved. 
+It is still possible that the original ID appears in the log files for the debugging purposes but log files are purged regularly. 
+The user ID is case sensitive.
 
-### Item Identifiers
+### Item identifiers
 
-As a customer or, more precisely, the specialty department you need to evaluate what should be presented as recommendations on your site. In the E-commerce business you mostly have the possibility to track items based on:
+People responsible for the sales policy in place in your organiation must decide what should be presented as recommendations on the website. 
+In the eCommerce business you mostly have the possibility to track items based on:
 
 - **Stock Keeping Unit** (SKU) or
 - **Master Item Identifier** (MII)
 
-The exact identifiers that are tracked will also be recommended ("*what-you-track-is-what-you-get"*). By default it is not possible to track SKUs but recommend MIIs. Following is an example of a typical decision case in the E-commerce business:
+The exact identifiers that are tracked are also recommended ("*what-you-track-is-what-you-get"*). 
+By default, it is not possible to track SKUs but recommend MIIs. 
```

Comment:
MII could use an explanation (SKU is used throughout commerce doc anyway nad is a bit clearer, I think)